### PR TITLE
Add more/indeed jobs

### PR DESCRIPTION
**Pull request by rkimera94**

### CODE REVIEW

File: database/things/http.pldb [hunk 1]
```diff
@@ -73,6 +73,8 @@ leachim6 HTTP
   httpd = socketserver.TCPServer(('', 8000), Handler)
   httpd.serve_forever()
   
+indeedJobs http developer
+ 2022 771996
```
Comment (celtic-coder) [inline database/things/http.pldb:77]: How do you determine the job requirements of a "http developer" ? It seems to be a very broad category.

File: database/things/mips.pldb [hunk 2]
```diff
@@ -62,8 +62,10 @@ rijuRepl https://riju.codes/mips
   	.data
   message:
   	.string "Hello, world!\n"
-  
 
+indeedJobs mips engineer
+ 2022 1150
```
Comment (celtic-coder) [inline database/things/mips.pldb:67]: MIPS is an old architecture. Again, might this be for another type of work?

File: database/things/owl.pldb [hunk 3]
```diff
@@ -89,6 +89,9 @@ linguistGrammarRepo https://github.com/textmate/xml.tmbundle
 
 hopl https://hopl.info/showlanguage.prx?exp=4934
 
+indeedJobs owl developer
+ 2022 3
```
Comment (celtic-coder) [inline database/things/owl.pldb:93]: OWL is an XML data format. How is this job number determined?

File: database/things/oz.pldb [hunk 4]
```diff
@@ -151,7 +151,9 @@ rijuRepl https://riju.codes/oz
       {System.showInfo 'Hello, world!'}
       {Application.exit 0}
   end
-  
+
+indeedJobs oz engineer
+ 2022 19
```
Comment (celtic-coder) [inline database/things/oz.pldb:156]: The latest stable release of Oz was in 2018 (https://en.wikipedia.org/wiki/Oz_%28programming_language%29). Might this number be too high?

File: database/things/rebol.pldb [hunk 5]
```diff
@@ -105,6 +105,9 @@ rijuRepl https://riju.codes/rebol
   
 tryItOnline rebol
 
+indeedJobs rebol developer
+ 2022 3
```
Comment (celtic-coder) [inline database/things/rebol.pldb:109]: I am surprised that there are any Rebol jobs! It's an extremely niche language.

File: database/things/rexx.pldb [hunk 6]
```diff
@@ -136,6 +136,9 @@ rijuRepl https://riju.codes/rexx
   
 tryItOnline rexx
 
+indeedJobs rexx engineer
+ 2022 325
```
Comment (celtic-coder) [inline database/things/rexx.pldb:140]: Again, I'm surprised that the number is so high. Are these jobs at IBM?

Comment (rkimera94) [reply database/things/rexx.pldb:140]: true, I think this should come to 40. 
Thanks for the feedback @celtic-coder  let me revise these values

Comment (rkimera94) [reply database/things/rexx.pldb:140]: not all jobs are at IBM. [link](https://www.indeed.com/jobs?q=rexx&l=Texas&vjk=5341201a75293683)

File: database/things/owl.pldb (hunk 3 above)
Comment (rkimera94) [reply database/things/owl.pldb:93]: okay, I had looked at owl developer! this has been fixed.

File: database/things/mips.pldb (hunk 2 above)
Comment (rkimera94) [reply database/things/mips.pldb:67]: I had looked at the MIPS architecture,  [Link](https://www.indeed.com/jobs?q=MIPS+architecture&l=Texas&vjk=0098240a434d3527)

File: database/things/http.pldb (hunk 1 above)
Comment (rkimera94) [reply database/things/http.pldb:77]: Here was looking at jobs that have HTTP as a requirement

Comment (rkimera94) [reply database/things/http.pldb:77]: its a very very broad category

**Comment by rkimera94** [comment]
@celtic-coder  I have made some changes to this PR. and also shared some links(way have been doing the search)

**Comment by breck7** [comment]
Great work @rkimera94 ! This is live now. Great review @celtic-coder !

**Pull request merged**